src/wallet/directWallet.ts

```
@@ -0,0 +1,1401 @@
+                        }
+                        break;
+                    case SettlementEventType.BatchStarted:
+                        if (step !== undefined) {
+                            continue;
+                        }
+                        const res = await this.handleBatchStartedEvent(
+                            event,
+                            intentId,
+                            this.arkServerPublicKey,
+                            this.forfeitOutputScript
+                        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)